.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -125,6 +117,14 @@ RUN owl-bot copy-code --version
 RUN chmod o+rx $(which owl-bot)
 RUN apk del -r npm && apk cache clean
 
+# Here we transfer gapic-generator-java from the previous stage.
```

Comment:
I moved the generator jar copy later in the final stage so the previous steps can run in parallel with `mvn install`. It saves dozens of seconds.